examples/sanic_aiomysql_with_global_pool.py

```diff
@@ -0,0 +1,63 @@
+# encoding: utf-8
+"""
+You need the aiomysql
+"""
+import asyncio
+import os
+
+import aiomysql
+import uvloop
+from sanic import Sanic
+from sanic.response import json
+
+database_name = os.environ['DATABASE_NAME']
+database_host = os.environ['DATABASE_HOST']
+database_user = os.environ['DATABASE_USER']
+database_password = os.environ['DATABASE_PASSWORD']
+app = Sanic()
+asyncio.set_event_loop_policy(uvloop.EventLoopPolicy())
+
+
+async def get_pool(*args, **kwargs):
```

Comment:
this can just take the loop and app (less obfuscated):  `async def get_pool(loop, app):`. Then use `app` instead of `args[0]` on line 34.